source/_integrations/openweathermap.markdown

```diff
@@ -25,7 +25,20 @@ There is currently support for the following device types within Home Assistant:
 - Sensor
 - Weather
 
-You need an API key, which is free, but requires a [registration](https://home.openweathermap.org/users/sign_up).
+You need an API key, it requires a [subscription](https://openweathermap.org/api/one-call-3). The subscription has a free tier with 1000 calls/day. Consider setting the limit on the OpenWeatherMap website to stay under the threshold where API usage incurs a cost.
+
+## ⚠️ Important Deprecation Notice
+
+### OpenWeatherMap API V2.5 Deprecation
+
+OpenWeatherMap API V2.5 will be closed in June 2024. After that you can use only API V3.0.
+
+You need to go to OpenWeatherMap website, activate One Call subscription.
```

Comment:
Without the `-`s suggested in https://github.com/home-assistant/home-assistant.io/pull/32609#discussion_r1613094040 this doesn't format very well

preview: https://deploy-preview-32609--home-assistant-docs.netlify.app/integrations/openweathermap/#openweathermap-api-v25-deprecation

Reply:
copy pasted from suggestion